### ISSUE TITLE
Request (99.99999% sure it's not possible)

### ISSUE THREAD

**Issue by Vitae666** [nls0/C-Hero-Calc#4]
What would be the chances that somehow the Calc would be able to see what you have and if you can't beat the Quest, it gives a suggestion of who you might need to level and to what level to be able to beat it?
Like say I input QuestXX-3 with what the usual suggestion is of like 5 - 10 heroes.
Happens that of those 10 heroes, auri:2 would actually be able to beat the lineup if it was auri:5

Or gives a suggestion of a hero that you don't have listed.
"If you had ailen:30 you could beat that line up with this setup:"

Guess what? I have ailen:60! So, there ya go!
I pop in that result and done :)
Or I set ailen:20 to be levelled up in the GUI to ailen:30, wait a few days and then done.

I know that this type of thing wouldn't help when it comes to the automation from the GUI, unless somehow it was added but kinda think that might be a bit harder to do because it would need to read the output and do it's stuff again. So, I'm not worried about the GUI.
But for the Calc, those rare people that might still use it in raw form, it might help.

BUT like I said, I don't think this would ever come in because along with the original calc, it would need to re-run multiple other calcs to see what it could output.
But damn, it would be glorious :)

**Comment by robgw84** [comment]
Actually what sort of works for that is to increase the max followers and run that, see what heroes of yours it puts in the lineup then reset the max to your current followers and slowly increase the level of the heroes from the lineup until the followers needed matches yours. I did that a while back and found if I leveled up Rigr to 20 I could beat a quest that I couldnt beat before.

Doing that also showed me that if I leveled up Master Lee to 35 it would drop the required followers for a quest around 100 from 35mil to 21mil.

**Comment by Vitae666** [comment]
Ok, sorry, bit thickheaded right now and not sure what you mean.
To check the minimum followers and what heroes/monsters are needed, I use -1 for the Max and enable all my heroes with their current levels.
For quest 95-3 I get:
Solution for   [quest95-3 | Followers: 126136000 | e20 e17 e17 a18 e17 e18 <==]:
[Followers: 51245600 | hunter:99 f20 lordkirk:69 f17 <==]

So, what do I increase? The levels of all my heroes and rerun it until I get something lower?

If I don't enable all my heroes I get
Solution for [quest95-3 | Followers: 126136000 | e20 e17 e17 a18 e17 e18 <==]:
[Followers: 51245600 | shygu:79 f20 lordkirk:69 f17 <==]
Not much diff, just a diff hero combo

And if I use the 110% Max setting, I really don't get a different result. Part of the reason I'm not sure what you mean :)

**Comment by robgw84** [comment]
You run it with your best heroes and no max, set the lower limit a little below your current max just to save some time, my current max is 25mil so I set it for 20mil. Then you run it for the quest you want to solve.

This will give you what number of followers you will need in order to beat it with your current heroes. You take one of the heroes that it gives you as a result (one that isnt max lvl of course) and raise its level by 5 and try to run it again to see if the followers needed drops down some. Keep raising the level until you hit max or until the number of followers drops to or close to what you have.

So say you had 5mil followers and running it with your heroes but no max tells you that you need 7mil followers and Athos which say is at level 10. You raise his level to 15 in the lineup, I dont mean you actually raise his level in the game, and run it again. Nothing changes so you run it a couple more times, finally when you have him raised to lvl 35 the followers drop to 5mil, then you know that if you get Athos to lvl 35 that you can beat the quest with your current followers.

If you get him to max lvl and the number of followers needed still doesnt change, then you can reset him to his actual level and try with another hero if there is one in the suggested lineup. If it doesnt work with that hero either then try raising them both up slowly and see if that works.

**Comment by Vitae666** [comment]
Ok. got it.
Actually had started to do it that way, after I thought more about it
Once I was at:
_[Followers: 51245600 | shygu:85 f20 lordkirk:75 f17 <==]_
it still wasn't giving me any results
Figured I'd try it with using and raising all my heroes to see if there would be any diff.
Put the heroes and their levels in Excel, had Excel add 1 level, copy/pasted it into gen.input and eventually came up with:
_[Followers: 33155600 | werewolf:99 f20 neptunius:82 lordkirk:72 <==]_

Calcs didn't really take much longer than it took when I was doing just shygu & lordkirk

Matter of fact, shygu:87 & lordkirk:77 works. Just checked.
BUT I only had to raise neptunius & lordkirk 3 levels each. werewolf was already 99
If I'd stayed with shygu & lordkirk it would have been **8 levels each**
Heck of a difference ;-)

**Comment by robgw84** [comment]
Heroes that reflect damage are really useful, I was stuck at around quest 70 until I got Master Lee from the Season Pass

**Comment by Vitae666** [comment]
Correction, Nep & Kirk worked at 81 & 71
Was looking at the wrong part of my Excel file :)
So, 2 levels each. But Shy/Kirk being 8 each is still correct.

Also, I realize that my method might not be the best because of using all the heroes. Would take longer for most calcs. I likely just got lucky.
Haven't tried others yet.

**Comment by robgw84** [comment]
Yeah thats why I run it with just my heroes and see what ones it suggests and try boosting them. It gives me an idea of what hero is useful for it without taking forever.

**Comment by Vitae666** [comment]
Sucks that you have to balance time vs cost of leveling.
On the one hand, doing 8 calcs with 2 heroes could be quicker than 5 calcs with 73 heroes, but the cost is the difference between 16 or 10 and running out of stuff to level heroes with and being stuck sooner at an impassable quest. :(
